scripts/create_perf_json.py

```diff
@@ -1156,16 +1156,12 @@ def fixup(form: str) -> str:
                             if events[name].unit.startswith('cpu') and name in form:
                                 if form == name or form.startswith(f'{name} '):
                                     form = f'{pmu_prefix}@{name}@' + form[len(name):]
```

Comment:
can we remove this too?

Reply:
Yes, I think so. This should be not required now. I will update it.